.github/workflows/sonarqube.yaml

```diff
@@ -0,0 +1,33 @@
+name: SonarQube
```

Comment:
we can push to sonarqube, but being this a public repo, we should push to codacy, so we can use the badges and so on
(just a note btw, not completely relevant here)